package.json

```diff
@@ -64,6 +63,7 @@
     "browserify": "^16.2.3",
     "chai": "^4.1.2",
     "core-js": "^3.6.4",
+    "credentials-context": "^1.0.0",
```

Comment:
I think this needs to be moved back into non-dev `dependencies` (since it's being used in `lib/contexts/index.js`).

Reply:
Really good catch: https://github.com/digitalbazaar/vc-js/pull/7/commits/c3488cb9b337d0d22ecb8c05ab98401d1df5e21a

Sorry about that.